languages/python/django-oso/Makefile

```diff
@@ -13,6 +13,10 @@ test-requirements: .make.test-requirements-install
 test: test-requirements
 	pytest tests
 
+test22: test-requirements
+	pip install django==2.2
```

Comment:
Good enough for now, but we should look into tox (or something similar) as a follow-up task

Reply:
Yep

Reply:
created follow-up task